spec/dsl_spec.rb

```diff
@@ -16,17 +16,25 @@
       an_object.step("foo").should == "foo"
     end
 
+    it 'dynamically creates the module name from the tag' do
+      context.steps_for(:dynamic_foo) do
+        step("foo") { "foo" }
+      end
+
+      defined?(DynamicFooSteps).should == 'constant'
+    end
+
     it 'remembers the name of the module' do
-      mod = context.steps_for(:foo) {}
-      mod.tag.should == :foo
+      mod = context.steps_for(:bar) {}
```

Comment:
This tags were changed because the test suite was trying to redefine `module Foo` for each test.